recipes/openmc-plotter/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "openmc-plotter" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/openmc-plotter-{{ version }}.tar.gz
+  sha256: 06e9e17f12f119e81b9fbf80cee9276cf3ec144a6be9f783b304646d357c32b4
+
+build:
+  entry_points:
+    - openmc-plotter=openmc_plotter.__main__:main
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: true  # [win or py<36]
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - matplotlib-base
+    - numpy
+    - openmc >0.12.2
+    - pyside2
+    - python
+    - ipython
+
+test:
+  imports:
+    - openmc_plotter
+  source_files:
+    - tests
```

Comment:
The tests folder isn't included in the PYPI source release, that's why conda-build can't copy it. Switch to the GitHub release url. https://pypi.org/project/openmc-plotter/#files

Reply:
Thank you!